### DISCUSSION TITLE
neodb/

### DISCUSSION THREAD

**Discussion by giscus** [eallion/eallion.com#105] [Giscus]
# neodb/

前言几个月之前在 长毛象联邦宇宙 里问过 NeoDB 官方有没有 API，得到肯定回答后，我就着手计划把观影页面的 API 搬到 NeoDB 了。前几天豆瓣的图片挂掉之后，加快了

https://www.eallion.com/neodb/

**Comment by lss53** [comment]
蜗牛大佬，根据您博客观影源码复现只差最后一步，图片不能显示，F12大法提示：lazy entered error

**Reply by eallion** [reply]
如果加载了 lazyload 插件，`<img>` 要包含 `data-src` 和 `src` ，`<img data-src="" src="">` ，有可能是这个原因。

**Comment by conge** [comment]
今天按照这个折腾了一下，
把自己页面的书影音数据源也切换到neodb了。

来表示感谢！

**Reply by eallion** [reply]
🙌🙌🙌
